### PR TITLE
feature: add new bool property withClasses that add classes to rows

### PR DESCRIPTION
**Pull request by MariannaMilovanova**

### CODE REVIEW

File: tests/index-test.js [hunk 1]
```diff
@@ -24,6 +24,8 @@ const rows = [
 ];
 const keyGetter = r => r.a;
 
+const withClasses = true;
```
Comment (jorrit) [inline tests/index-test.js]: Please move this line into the test function.

File: src/index.js [hunk 2]
```diff
@@ -26,6 +26,14 @@ const getClassNameOrStyleProps = (classNameOrStyle, componentState) => {
   return {};
 };
 
+function headerClass(withClasses, key) {
+  return withClasses ? { className: `th-${key}` } : {};
```
Comment (jorrit) [inline src/index.js]: I think `header-${key}` is more descriptive than `th-${key}`.

File: src/index.js [hunk 3]
```diff
@@ -26,6 +26,14 @@ const getClassNameOrStyleProps = (classNameOrStyle, componentState) => {
...
+}
+
+function rowClass(withClasses, key) {
+  return withClasses ? { className: `tr-${key}` } : {};
+}
+
```
Comment (jorrit) [inline src/index.js]: I think `row-${key}` is more descriptive than `row-${key}`.

File: README.md [hunk 4]
```diff
@@ -126,6 +126,7 @@ Properties marked with an asterisk (`*`) are required.
 | | `object` | CSS styles to apply to the parent table tag. |
 | | `function` | Function returning one of the above. The function receives a state object with boolean property narrow indicating if the current presentation is narrow or wide. |
 | `initialNarrow` | `bool` | Initially render the table in narrow mode when rendering serverside. Set to true when you expect the browser to be narrow to prevent rerendering client side. |
+| `withClasses` | `bool` | Add classNames to each rows and headers cells by using keyGetter function. |
```
Comment (jorrit) [inline README.md]: Please change to 'Add unique class names to each row and header cell, respectively `row-KEY` and `header-KEY`.'.

**Review by jorrit — CHANGES_REQUESTED** [review]
Thanks for your PR, Marianna. Could you take a look at these review comments and address them? Thanks!

**Comment by MariannaMilovanova** [comment]
@jorrit please see my changes

**Review by jorrit — APPROVED** [review]

**Comment by jorrit** [comment]
My apologies for the delay. 0.6.0 has been released with this feature. Thanks for your contribution!

**Comment by MariannaMilovanova** [comment]
Thank you!

**Pull request merged**